sched/job_def_serializer_property_test.go

```diff
@@ -15,14 +16,32 @@ func Test_RandomSerializerDeserializer(t *testing.T) {
 
 	properties.Property("Serialize JobDef", prop.ForAll(
 		func(job *Job) bool {
-			ValidateSerialization(job, false, t)
-			ValidateSerialization(job, true, t)
+			serializeBinary := ValidateSerialization(job, false)
```

Comment:
nit: there are a few places in this PR where you can combine 'if' and the line doing assignment.